api/agent/slots.go

```diff
@@ -352,6 +354,11 @@ func getSlotQueueKey(call *call) string {
 		hash.Write(unsafeBytes("\x00"))
 	}
 
+	if driver != nil {
+		slotExtns := driver.GetSlotKeyExtensions(call.Extensions())
+		hash.Write(unsafeBytes(slotExtns))
```

Comment:
Is a `hash.Write(unsafeBytes("\x00"))` needed after this line?

Reply:
Missed that. Updated the change